config/test.json

```diff
@@ -1,10 +1,15 @@
 {
   "db": {
-    "host": "postgres",
+    "host": "localhost",
     "schema": "polygon_parts",
     "username": "postgres",
-    "password": "1234",
-    "database": "postgres",
-    "enableSslAuth": false
+    "password": "postgres",
+    "database": "test",
+    "enableSslAuth": false,
+    "logging": false,
+    "entities": ["src/**/DAL/*.ts"],
```

Comment:
test.json is used in the test environment. tests are run by jest and transpiled to JS by ts-jest internally (and dynamically once they are imported). see reference from [docs](https://jestjs.io/docs/code-transformation)

> _Jest passes files through code transformation on demand (for instance when a require or import is evaluated)_

TypeORM is [importing](https://github.com/typeorm/typeorm/blob/db508424c828cd9b482f212099da3ee29c544565/src/util/ImportUtils.ts#L8) the files found by the glob pattern in test.json

changing to `["src/**/DAL/*.js"]` is failing, since no files with .js extension are generated.